tests/test_cli.py

```diff
@@ -102,5 +113,22 @@ def test_database_for_unsupported_song(self):
 		"""
 		self.assertEqual(get_lyrics("Bitch Lasagna", "Party in Backyard")[:7], "[Intro]")
 
+	@mock.patch('requests.post', return_value=R())
+	def test_that_get_lyrics_does_not_break_with_request_giving_wrong_status_code(self, mock_requests):
+		"""
+		Test the get_lyrics does not break with requests giving wrong status code
+		"""
+		self.assertEqual(get_lyrics("Ki", "Ki", True), "Couldn\'t get lyrics for Ki by Ki.\n")
+
+	@mock.patch('requests.post', side_effect=requests.exceptions.RequestException)
+	def test_that_get_lyrics_do_not_break_with_error_in_request(self, mock_requests):
+		"""
+		Test the get_lyrics does not break with error in requests
+		"""
+		self.assertEqual(get_lyrics("Ki", "Ki", True), "Couldn\'t get lyrics for Ki by Ki.\n")
+
+	@mock.patch('requests.post', return_value=R(200, "Season 3 is supernatural"))
+	def test_that_get_lyrics_calls_requests(self, mock):
+		self.assertEqual(get_lyrics("River", "Dale", True), "Couldn't get lyrics for River by Dale.\nSeason 3 is supernatural")
```

Comment:
this needs work, please correct the equality statement

Reply:
It seems correct to me